docs/client/full-api/api/templates.md

```diff
@@ -264,12 +264,10 @@ Another example where the subscription depends on the data context:
 
 ```js
 Template.comments.onCreated(function () {
-  var self = this;
-
-  // Use self.subscribe with the data context reactively
-  self.autorun(function () {
+  // Use this.subscribe with the data context reactively
+  this.autorun( ()=> {
```

Comment:
The spacing should be like:

```js
this.autorun(() => {
```

Since these are "official" code samples, I think it's fair to be nitpicky in this way.